proto/job.proto

```diff
@@ -0,0 +1,101 @@
+// The file defines the CommandService API.
+syntax = "proto3";
+
+package job;
+
+option go_package = "/job";
+
+import "google/protobuf/timestamp.proto";
+
+// JobService provides methods for starting a job and streaming its output.
+service JobService {
+  // Starts the job and returns an identifier for the job.
+  rpc StartJob(StartRequest) returns (StartResponse) {}
+  // Returns a list of running jobs with their jobId and jobStart timestamps.
+  rpc ListJobs(ListJobsRequest) returns (ListJobsResponse) {}
+  // Stream the output of the job.
+  rpc StreamJobOutput(JobIdentifier) returns (stream JobOutput) {}
+  // Stops a running job.
+  rpc StopJob(JobIdentifier) returns (StopResponse) {}
+}
+
+// StartRequest contains the job command and rootFS to execute the command in.
+message StartRequest {
+  // The job command line to run.
+  string jobCommand = 1;
```

Comment:
Recommend following the style guide for proto files: https://protobuf.dev/programming-guides/style/

Reply:
Sure. I'll check it out and reformat.

Reply:
Reformatted message field names, updated comments, and placed it inline with design-document.md.